notebooks/basic-usage.ipynb

```diff
@@ -0,0 +1,310 @@
+{
```

Comment:
<p>We should probably change that behavior and just return an arviz trace object with no reference to the tensors.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc4/pull/190/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>That probably should be a later PR, out of scope for this one?</p>